include/openssl/ssl3.h

```diff
@@ -25,6 +25,9 @@
 extern "C" {
 #endif
 
+/* Flag used on OpenSSL ciphersuite ids to indicate they are for SSLv3+ */
+# define SSL3_CK_CIPHERSUITE_FLAG                0x03000000
```

Comment:
I'm not saying change the ID values.  Just the flag.  If you still think that is an issue, fine.